lib/src/interactions.dart

```diff
@@ -66,12 +66,12 @@ abstract class IInteractions {
   Future<void> deleteGuildCommands(List<Snowflake> guildIds);
 
   /// Fetches all global bots command
-  Stream<ISlashCommand> fetchGlobalCommands();
+  Stream<ISlashCommand> fetchGlobalCommands({bool withLocales = false});
```

Comment:
I think it'd be less confusing if the default was `true` instead of `false` for fetching commands.